aztk/spark/models/plugins/resource_monitor/configuration.py

```diff
@@ -0,0 +1,30 @@
+import os
+from aztk.models.plugins.plugin_configuration import PluginConfiguration, PluginPort, PluginTarget, PluginTargetRole
+from aztk.models.plugins.plugin_file import PluginFile
+from aztk.utils import constants
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+class ResourceMonitorPlugin(PluginConfiguration):
+    def __init__(self):
+        super().__init__(
+            name="resource_monitor",
+            ports=[
+                PluginPort(
+                    internal=8086,
+                    public=True,
```

Comment:
Is there a reason this is open on ssh? Since this is just the influx api endpoint and not a UI, why are we opening it?